src/tools/run-make-support/src/cc.rs

```diff
@@ -27,11 +28,11 @@ impl Cc {
     /// WARNING: This means that what flags are accepted by the underlying C compile is
     /// platform- AND compiler-specific. Consult the relevant docs for `gcc`, `clang` and `mvsc`.
     pub fn new() -> Self {
-        let compiler = env::var("CC").unwrap();
+        let compiler = env_var("CC");
```

Comment:
Discussion (non-blocking): this is completely stylistic, but should we have this be `run_make_support::env::var()`?

Reply:
Not a big fan because `use std::env` is often used in both `run-make-support` lib and `run-make` tests, making this import ambiguous. Hence why I picked `env_var` and `env_var_os`. Does it make sense to you?

Reply:
Yep it does, I was just thinking of possibilities, what you said makes a lot of sense 👍